bokehjs/test/unit/core/bokeh_events.ts

```diff
@@ -61,3 +64,88 @@ describe("BokehEvent", () => {
     expect(event.values).to.be.equal({target: widget, action: "do_that"})
   })
 })
+
+describe("AxisClick event", () => {
+  for (const {side, pt} of [{side: "left", pt: xy(5, 200)}, {side: "right", pt: xy(395, 200)}, {side: "above", pt: xy(200, 5)}, {side: "below", pt: xy(200, 395)}]) {
+    it(`should support linear axes on side: ${side}`, async () => {
+      const plot = new Plot({
+        width: 400,
+        height: 400,
+        title: null,
+        toolbar_location: null,
+        x_range: new Range1d({start: 0, end: 10}),
+        y_range: new Range1d({start: 0, end: 10}),
+      })
+      const axis = new LinearAxis()
+      plot.add_layout(axis, side as any)
```

Comment:
This `as any` can be avoided in a number of ways. Either use `side: "left" as const` for all sides, which narrows the type to `"left" | "right" | "above" | "below"`. Alternatively you could use `statisfies` operator on the array with an appropriate type (`{side: Side, pt: Point}[]`). Alternatively and preferably rewrite this for loop into a function and multiple `it()` tests. The last one is my preferred approach, as it makes debugging of tests much easier.

Reply:
I will push changes to this PR to set the example of the preferred approach.